LayoutTests/fast/events/no-mousemove-on-modifier-key-press.html

```diff
@@ -0,0 +1,43 @@
+<html>
+<head>
+<script src="../../resources/js-test.js"></script>
+<script src="../../resources/ui-helper.js"></script>
+<style>
+.hidden {
+    display: none;
+}
+</style>
+</head>
+
+<body>
+<p id="description"></p>
+<p id="manual-instructions" class="hide">To run this test by hand, press a modifier key.</p>
+<div id="console"></div>
+<script>
+window.jsTestIsAsync = true;
+
+async function runTest()
+{
+    if (!window.testRunner)
+        document.getElementById("manual-instructions").classList.remove("hidden");
+
+    function fireKeyDownEventsWithModifiers() {
+        ['metaKey', 'altKey', 'shiftKey', 'ctrlKey'].forEach(modifierKey => {
+            UIHelper.keyDown("k", [modifierKey]);
+            debug('Dispatched KeyDown event with modifier ' + modifierKey);
+        });
+    }
```

Comment:
`UIHelper.keyDown` is asynchronous on iOS, where we use `UIScriptController`. We should probably await these calls:

```suggestion
    async function fireKeyDownEventsWithModifiers() {
        ['metaKey', 'altKey', 'shiftKey', 'ctrlKey'].forEach(modifierKey => {
            await UIHelper.keyDown("k", [modifierKey]);
            debug('Dispatched KeyDown event with modifier ' + modifierKey);
        });
    }
```